src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+const initState = {
+    products : [] 
+}
+
+const addProducts = (state = initState , action ) => {
```

Comment:
In production applications, you will have multiple reducers. 
There should be separate file for each reducer and the index.js should import all the reducers and combine them using combineReducers